### PR TITLE
LoginPage validation update

### PR DESCRIPTION
**Pull request by HardCoder404**
# Description : 
### In this bug, I just added a validation in Login Page.

# ScreenShot : 
Screenshot 2024-05-13 201921
Screenshot 2024-05-13 201937

### CODE REVIEW

**Comment by rishicds** [comment]
Hi @HardCoder404 , please fix the merge conflicts, it will be merged after that, also mention which issue this closes.

**Comment by HardCoder404** [comment]
The conflict has been resolved. Please review and merged it.

**Review by codewarnab — CHANGES_REQUESTED** [review]
sorry to bother you again please move the validation inside the login function which is in the userAuthService.js it will maintain the code quality .
also their is already email validatory so no need to valid it two times

File: src/pages/user/UserRegistration.jsx [hunk 1]
```diff
@@ -51,6 +51,23 @@ function LoginRegisterForm() {
...
+      return;
+    }
+    if (!password) {
+      setError("Please enter a valid password");
+      return;
+    }
+    login(email, password, setError, setButtonState);
+  };
+
+=======
```
Comment (codewarnab) [inline src/pages/user/UserRegistration.jsx:70]: remove this thing also

**Comment by codewarnab** [comment]
@HardCoder404 its been 10 days give update or we  have to close the pr

**Pull request closed without merging**